src/views/Releases/ListReleases/index.jsx

```diff
@@ -230,10 +262,20 @@ function ListReleases(props) {
     return { name: release.name, new_data_version: data.data.new_data_version };
   };
 
+  const handleReadOnlySubmit = async () => {
+    const release = dialogState.item;
+
+    if (release.read_only && requiresSignoffs(release)) {
+      return scheduleReadWriteChange(release);
+    }
+
+    return updateReadonlyFlag(release);
+  };
+
   const handleReadOnlyComplete = result => {
     setReleases(
       releases.map(r => {
-        if (r.name !== result.name) {
+        if (r.name !== result.name || (r.read_only && requiresSignoffs(r))) {
           return r;
         }
 
```

Comment:
Just below here is where you'll need to add the `scheduledChange` to the Release.